### ISSUE TITLE
Scaler in pipeline not fitting

### ISSUE THREAD

**Issue by mtanghu** [creme-ml/creme#359]
## Versions

**`creme` version** : 
0.6.1
**Python version** : 
3.6.9
**Operating System** :
Windows 10

## Describe the bug

It looks like creating a pipeline with a StandardScaler() doesn't fit the StandardScaler for me. When I use the debug_one() it seems that the scaler always scales to 0.

## Steps/Code to Reproduce

Running this sample shows the issue I'm having with the scaler.

```python
from creme import preprocessing as creme_preprocessing
from creme import linear_model

scaled_model = creme_preprocessing.StandardScaler() | linear_model.LinearRegression()

for i in range(1000):
    scaled_model.fit_one({"i": i}, i)
    
print(scaled_model.debug_one({"a": 0}))
print(scaled_model.debug_one({"a": 1000}))
````
produces:

```
0. Input
--------
a: 0 (int)

1. StandardScaler
-----------------
a: 0.00000 (float)

2. LinearRegression
-------------------
Name        Value      Weight       Contribution  
Intercept    1.00000    950.00000      950.00000  
        a    0.00000      0.00000        0.00000  

Prediction: 950.00000
0. Input
--------
a: 1000 (int)

1. StandardScaler
-----------------
a: 0.00000 (float)

2. LinearRegression
-------------------
Name        Value      Weight       Contribution  
Intercept    1.00000    950.00000      950.00000  
        a    0.00000      0.00000        0.00000  

Prediction: 950.00000
```

**Comment by MaxHalford** [comment]
This is not a bug, it's a feature :)

The unsupervised parts of a pipeline are fitted when `predict_one` is called. Basically, this is the optimal behaviour. Performance would be much lower if we updated the unsupervised parts only when the model is trained. Indeed, if the data is available during `predict_one`, then why wouldn't we updated the unsupervised part of the pipeline there and then?

If you think about it, it's weird to train a pipeline without making predictions before anyway. The only case where you would to do that is pre-training (warming up). But even in that case you would want to be making predictions to know the performance of your model.

I've thought about this long and hard and I keep thinking about this. I'm happy to discuss alternative solutions.
